docs/en/Time-To-Live.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: global
+title: Time to Live
+nickname: Time to Live
+group: Features
+priority: 8
+---
+
+* Table of Contents
+{:toc}
+
+Alluxio supports a `Time to Live (TTL)` setting on each file and directory in the namespace. This
+feature can be used to effectively manage the Alluxio cache, especially in environments with strict
+guarantees on the data access patterns. For example, if analytics is only done on the last week of
+ingested data, TTL can be used to explicitly flush old data to free the cache for new
+files.
+
+## Overview
+
+Alluxio has TTL attributes associated with each inode. These attributes are journaled and persist
+across cluster restarts. The active master node is responsible for holding the metadata in memory
+when Alluxio is serving. Internally, the master runs a background thread which periodically checks
+if files have reached their TTL.
+
+Note that the background thread runs on a configurable period, by default 1 hour. This means a TTL
+might not be enforced until the next check interval, ie. short TTL intervals can take up to
+two hours to be enforced. The interval length is set by the `alluxio.master.ttl.checker.interval`
+property.
+
+For example, to set the interval to 10 minutes, add the following to alluxio-site.properties:
+
+```
+alluxio.master.ttl.checker.interval=10m
+```
+
+Refer to the [configuration page](Configuration-Settings.html) for more details on setting Alluxio
+configurations.
+
+While the master node enforces TTLs, it is up to the clients to set the appropriate TTLs.
+
+## APIs
+
+There are three ways to set the TTL of a path.
+
+1. Through the Alluxio shell command line.
+1. Through the Alluxio Java File System API.
+1. Passively on each load metadata or create file.
```

Comment:
Add a **NOTE** to mention that the third way is user level which means all files loaded or created are with the same ttl specified by user, to achieve a more fine-grained ttl control on each file should use API.

Reply:
Updated, thanks.